### PR TITLE
Grammar and spelling fixes

### PR DESCRIPTION
**Pull request by isochronous**
I found a few more grammatical and word choice errors.  If you like, I can go through the whole Lexica Botania - I'd do it now, but I'm at work and can't spend that much time on it at the moment.  I'm a fairly good editor, and would enjoy contributing something back to Botania, so if you're interested just let me know.

### CODE REVIEW

File: src/main/resources/assets/botania/lang/en_US.lang [hunk 1]
```diff
@@ -2870,9 +2870,9 @@ botania.page.felPumpkin1=Nerfing warlocks again
 # -- MANA BLASTER
 botania.entry.manaBlaster=Mana Blaster
 botania.tagline.manaBlaster=Portable Mana Spreader, with lenses!
-botania.page.manaBlaster0=The &1Mana Blaster&0 is like a portable &1Mana Spreader&0, while shaped like a pistol for easy handling, it's not exactly a firearm, as much as it is just a portable spreader.<br>By utilizing &4Mana&0 from &4Mana carrying items&0 in the holder's inventory, it can shoot a mana burst when one is looking at.
+botania.page.manaBlaster0=The &1Mana Blaster&0 is like a portable &1Mana Spreader&0, while shaped like a pistol for easy handling, it's not exactly a firearm, as much as it is just a portable spreader.<br>By utilizing &4Mana&0 from &4Mana carrying items&0 in the holder's inventory, it can shoot a mana burst when one is looking at anything that can receive mana.
```
Comment (TheWhiteWolves) [inline src/main/resources/assets/botania/lang/en_US.lang]: Wire's right this is incorrect

**Comment by TheWhiteWolves** [comment]
Also you need to check your changes in game to make sure that the formatting is still ok and you haven't messed any of it up

**Comment by isochronous** [comment]
Okay, I'll see if I can set up gradle and verify those changes in-game.

**Comment by yrsegal** [comment]
you should probably squash (`git rebase`) your changes into one commit.

**Comment by TheWhiteWolves** [comment]
I can't see anything else wrong with it atm so as wire said rebase it and I think we'll be good to go.

**Comment by isochronous** [comment]
And I've somehow managed to just double the number of commits. Stand by, trying to fix.

**Comment by isochronous** [comment]
I've somehow gotten this branch into a really weird state that I don't know how to fix, so I'm just going to close the pull request and open another one with a single commit.

**Comment by isochronous** [comment]
Continued in PR https://github.com/Vazkii/Botania/pull/1798

**Pull request closed without merging**